### ISSUE TITLE
cropped the training images to 256*256, i met some errors

### ISSUE THREAD

**Issue by wzr0108** [vanvalenlab/deepcell-tf#635]
I cropped the training images so that the length and width are both 256, and the original image will become 4 small images, but the result of this training is completely wrong. I took the valid set to train again, and performed well on the test set, which proves that my training code is correct. The strangest thing is that I took the upper left corner of the training set picture for training, and the test result was good, but I took the upper right corner for training, but the result was very poor

**Comment by wzr0108** [comment]
![屏幕截图 2022-12-28 133758](https://user-images.githubusercontent.com/98294772/209763334-32ef5d74-b378-4301-a1b5-b873323cccc3.png)

**Comment by wzr0108** [comment]
I found a solution. I set the norm_method to "std" before and changed it to "whole_image". I would like to ask what is the difference between the two.
